utils/mirror_files.py

```diff
@@ -13,14 +12,7 @@ def copy_files(source_destination_tuples):
     for source_path, destination_path in source_destination_tuples:
         if settings.LOG_START_AND_END_COPYING_FILES:
             web_logger.error('Started copying file %s to %s' % (source_path, destination_path))
```

Comment:
@ffont I noticed this when making the change - this notification shouldn't be an error, I think it makes more sense to be info or debug.